### ISSUE TITLE
missing "(" , ")" in 8.4 Applying expressions

### ISSUE THREAD

**Issue by pywugate** [JuliaEarth/GeoStats.jl#409]
`box(point) = Box(point - Vec(2.5, 2.5, 2.5), point + Vec(2.5, 2.5, 2.5))`
fails
`box(point) = Box((point - Vec(2.5, 2.5, 2.5)), (point + Vec(2.5, 2.5, 2.5)))`
works

I think "()" are missing for both first and second arguments in Box()

**Comment by juliohm** [comment]
Can you elaborate why do you think this? The example in the book is working: https://juliaearth.github.io/geospatial-data-science-with-julia/08-splitcombine.html#applying-expressions

**Comment by pywugate** [comment]
it works now, maybe close it, I'm sorry.

I tried 4 times, restart Julia and ran this two lines from beginning of section 8
`box(point) = Box(point - Vec(2.5, 2.5, 2.5), point + Vec(2.5, 2.5, 2.5))`
`box.(gt.geometry)`

first time it gave me no method for Box()

second time same error, so I searched Box() syntax in section 4 and added "()" for both 1st and 2nd arguments.

third and fourth time I don't need to add "()".
